EnglishTranslation-master/QUEST_LV_0200.tsv

```diff
@@ -7601,14 +7601,14 @@ QUEST_LV_0200_20150714_007604	There's a saying that geniuses are freaks. How sho
 QUEST_LV_0200_20150714_007605	I want to request to you to bring the Metal Plate on Ramive Hills.{nl}There are many plates that I could not bring due to the monsters. You can do it right?
 QUEST_LV_0200_20150714_007606	This place is too isolated so the monsters rush in.{nl}Be careful all the time.
 QUEST_LV_0200_20150714_007607	Good!{nl}But, the status is too bad so it would take a long time...
-QUEST_LV_0200_20150714_007608	What a view here right?{nl}But, more amazing things are hidden. The records of the historian, Demetrius.{nl}
+QUEST_LV_0200_20150714_007608	What a view here right?{nl}But, more amazing things are hidden. The records of historian Demetrius.{nl}
```

Comment:
I'd avoid but's as they are in a lot of dialogue (because the Korean text always use 하지만.)

For variety's sake:
What a view, don't you think?{nl}Though there are grander things hidden here, such as the records of historian Demetrius.